shell/platform/android/io/flutter/plugin/common/MethodCall.java

```diff
@@ -15,6 +18,10 @@
 
     /**
      * Arguments for the call.
+     *
+     * Consider using {@link #arguments()} for cases where a particular run-time type is expected.
+     * Consider using {@link #argument(String)} when that run-time type is a {@link java.util.Map}
```

Comment:
Nit: no need to fully qualify `Map` and `JSONObject` in the docs since you import them.

(and in `#argument(String)`)